src/test/groovy/net/fabricmc/loom/test/util/ZipTestUtils.groovy

```diff
@@ -49,10 +49,16 @@ class ZipTestUtils {
         return file
     }
 
-    static String manifest(String key, String value) {
+	static String manifest(String key, String value) {
+		return manifest(Map.of(key, value))
+	}
+
+    static String manifest(Map<String, String> entries) {
```

Comment:
Mixed spaces and tabs 😍

We should probably figure something out for the inconsistent indentation in Groovy files, some files have spaces and others have tabs.

Reply:
Yeah, fixed this here. I will resolve this for all files in another PR later.

Reply:
See: https://github.com/FabricMC/fabric-loom/pull/850